integration_tests/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM gcr.io/google-appengine/python
+
+ENV PATH=$PATH:/builder/google-cloud-sdk/bin/
+
+RUN apt-get update && \
+    apt-get install -y --force-yes wget unzip ca-certificates git && \
+    # Setup Google Cloud SDK (latest)
+    wget -nv https://dl.google.com/dl/cloudsdk/release/google-cloud-sdk.zip && \
+    mkdir -p /builder && \
+    unzip -qq google-cloud-sdk.zip -d /builder && \
+    rm google-cloud-sdk.zip && \
+    /builder/google-cloud-sdk/install.sh --usage-reporting=false \
+        --bash-completion=false \
+        --disable-installation-options && \
+    # Install alpha and beta components
+    /builder/google-cloud-sdk/bin/gcloud -q components install alpha beta && \
+    /builder/google-cloud-sdk/bin/gcloud -q components update
+
+RUN apt-get install -y --force-yes python-dev && \
```

Comment:
So removing wget and unzip in a different RUN like won't actually save any space in the image. The layers still contain the files, they just get hidden. If you really want to remove these you'll have to do it all in one RUN line.